apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.js

```diff
@@ -90,31 +88,41 @@ async function sendNotifications (context, newsItem, taskId) {
         ],
     }), {})
 
-    const { keystone: contextMessage } = await getSchemaCtx('Message')
+    const { keystone: contextMessage } = getSchemaCtx('Message')
     for (const resident of residentsData) {
-        await sendMessage(contextMessage, {
-            ...DV_SENDER,
-            to: { user: { id: resident.user.id } },
-            lang: conf.DEFAULT_LOCALE,
-            type: defineMessageType(newsItem),
-            meta: {
-                dv: 1,
-                title: truncate(newsItem.title, { length: TITLE_MAX_LEN, separator: ' ', omission: '...' }),
-                body: truncate(newsItem.body, { length: BODY_MAX_LEN, separator: ' ', omission: '...' }),
-                data: {
-                    newsItemId: newsItem.id,
-                    organizationId: newsItem.organization.id,
-                    userId: resident.user.id,
-                    residentId: resident.id,
-                    userRelatedResidentsIds: get(residentsIdsByUser, resident.user.id, []).join(','),
-                    url: `${conf.SERVER_URL}/newsItem/${newsItem.id}`,
-                    validBefore: get(newsItem, 'validBefore', null),
-                    // The first truthy value will be returned, or null if no values are found.
-                    dateCreated: ['sendAt', 'publishedAt', 'updatedAt', 'createdAt'].reduce((result, field) => (result || get(newsItem, field)), null),
+        try {
+            await sendMessage(contextMessage, {
+                ...DV_SENDER,
+                to: { user: { id: resident.user.id } },
+                lang: conf.DEFAULT_LOCALE,
+                type: defineMessageType(newsItem),
+                meta: {
+                    dv: 1,
+                    title: truncate(newsItem.title, { length: TITLE_MAX_LEN, separator: ' ', omission: '...' }),
```

Comment:
It might be better to use deidcated `…` char here, since it `might` (im not sure) take less space in the resulting message

Need to check whether the target system support the encodings though

Reply:
I think there is no need to change something that already works well :)